pkgs/development/compilers/symbiflow-yosys-plugins/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchFromGitHub
+, symbiflow-yosys
+, zlib
+, readline
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "symbiflow-yosys-plugins";
+  version = "1.0.0.7-0060-g7454cd6b";
+
+  src = fetchFromGitHub {
+    owner  = "SymbiFlow";
+    repo   = "yosys-symbiflow-plugins";
+    rev    = "7454cd6b5e4fd22854e2ada219a5e3c3a06e0717";
+    sha256 = "0r9r31p7fy4ylfrwvwlbivq5a03xrph34blxbxzx2c8bc02mbv0s";
+  };
+
+  patches = [
+    ./make.patch
```

Comment:
@Mic92 is right this patch is not needed. Only `:=` assignments are not overridable in make.